docs/pages/guides/authentication.mdx

```diff
@@ -127,6 +128,115 @@ export default function App() {
 </Box>
 {/* End IdentityServer 4 */}
 
+<Box
+  name="Asgardeo"
+  createUrl="https://wso2.com/asgardeo/docs/guides/#developer-guide"
+  image={ASSETS.asgardeo}
+>
+
+| Website                       | Provider | PKCE      | Auto Discovery |
+| ----------------------------- | -------- | --------- | -------------- |
+| [Get Your Config][c-asgardeo] | OpenID   | Supported | Available      |
+
+[c-asgardeo]: https://console.asgardeo.io/
+
+- Make sure to check `Public Client` in the console.
+- Check `Implicit` in the Allowed grant types section.
```

Comment:
why do we need `Implicit` grant?

Reply:
Auth code flow only returns the `authorization_code` and the developer has to get the `access_token` by writing another method. Implicit flow directly returns the `access_token`. However, I separated them and added samples for both.